docs/guides/web/php.md

```diff
@@ -118,24 +192,23 @@ To serve PHP pages in CGI mode, you must install the apache server, configure it
 
 * Installation:
 
-```bash
-sudo dnf install httpd
-```
+	```bash
```

Comment:
You have many examples of unnecessary whitespace throughout this PR. This shouldn't be here.

Reply:
> You have many examples of unnecessary whitespace throughout this PR. This shouldn't be here.

Actually, @neelchauhan, he is trying to implement this: https://docs.rockylinux.org/guides/contribute/rockydocs_formatting/#tabbed-content-within-a-document

Which does require the four spaces he is showing. There's an issue with the tabs though, that I've outlined in another comment.